Resources/public/js/models/ez-contentmodel.js

```diff
@@ -316,6 +317,17 @@ YUI.add('ez-contentmodel', function (Y) {
              */
             relations: {
                 value: {}
+            },
+
+            /**
+             * The version info of the content
+             *
+             * @attribute versionInfo
+             * @type Object
+             * @default {}
+             */
+            versionInfo: {
```

Comment:
and in addition, I would add a getter function on that attribute so that `get('currentVersion`) returns an eZ.Version object.